cognitive/src/main/scala/com/microsoft/azure/synapse/ml/geospatial/AzureMapsBase.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.geospatial
+
+import com.microsoft.azure.synapse.ml.cognitive._
+
+abstract class AzureMapsBase(override val uid: String)
+  extends CognitiveServicesBase(uid) with HasSubscriptionKey with HasInternalJsonOutputParser {
```

Comment:
These classes seem a bit thin right now, perhaps the traits could be added into your actual models

Reply:
Done.. Removed these..